miniapp/USERGUIDE.md

```diff
@@ -538,12 +571,6 @@ In Host App, we can get the downloaded manifest information as following:
 
 HostApp can compare between the `downloadedManifest` and the latest manifest by `MiniApp.getMiniAppManifest` to detect any new changes.
 
-### Send message to contacts
```

Comment:
I think we should keep this section and have the links to the API Docs for each method. 

Maybe we should also have a brief explanation of what each messaging method is used for and explain that they should show a contact picker UI?

Reply:
You can take inspiration on the iOS PR for this @khairul-alam-licon

Reply:
True, I have updated the user guide.